src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -312,16 +311,7 @@ protected void onStartup() {
     private void validateSequenceDictionaries() {
         final SAMSequenceDictionary refDict = hasReference() ? reference.getSequenceDictionary() : null;
         final SAMSequenceDictionary readDict = hasReads() ? reads.getSequenceDictionary() : null;
-        List<SAMSequenceDictionary> variantDicts = new ArrayList<>();
-        if (hasFeatures()){
-            List<VCFHeader> variantHeaders = features.getAllVariantHeaders();
-            for (VCFHeader header : variantHeaders) {
-                SAMSequenceDictionary headerDict = header.getSequenceDictionary();
-                if (headerDict != null) {
-                    variantDicts.add(headerDict);
-                }
-            }
-        }
+        final List<SAMSequenceDictionary> variantDicts = hasFeatures() ? features.getAllSequenceDictionaries() : Collections.emptyList();
```

Comment:
Directly conflicts with https://github.com/broadinstitute/gatk/pull/1424 -- what do you intend as the final version of this line after both branches are merged?